onyo/lib/commands.py

```diff
@@ -262,12 +259,10 @@ def onyo_mkdir(inventory: Inventory,
         ui.print(line)
     if ui.request_user_response("Save changes? No discards all changes. (y/n) "):
         if not message:
-            prefix = "mkdir: "
-            # TODO: This does not actually account for implicitly created dirs (and therefore duplicates).
-            #       However, not mentioning implicit ones separately may be an advantage for a message subject
-            message = prefix + f"{', '.join(p.relative_to(inventory.root).as_posix() for p in dirs)}"
-            if len(message) > 80:
-                message = prefix + f" {', '.join(p.name for p in dirs)}"
+            new_dirs = set([Path(line[1:]) for line in inventory.diff()])
```

Comment:
This is not a reliable approach to get the paths. They should come from `Inventory.operations`. I'll push into this PR accordingly.

Reply:
This was solved by your own push, and my adjustments to the other functions.